spongebob-cli

```diff
@@ -79,21 +57,37 @@ def Play(DirectLink):
     except subprocess.CalledProcessError:
         print(colored("\nAn error occured while tying to play the video! Make sure you have mpv or youtube-dl installed.", "red"))
     
-    except KeyboardInterrupt:
-        print(colored("\nUser interruped with the operation, aborting.", "red"))
-        sys.exit()
-
+    
+@Halo(text="Downloading episode.", spinner="shark", color="green")
 def Download(source):
     try:
-        subprocess.run(["youtube-dl", source]) 
+        subprocess.run(["youtube-dl", source],
+                stdout=subprocess.DEVNULL,
```

Comment:
The reason i removed the voiding of the output is so people can see the download speed.